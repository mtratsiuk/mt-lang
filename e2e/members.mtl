(def square
  (|x|
    (* x x)))

(print
  [1 2 42]/2)

(print
  ((([2 nil 4 6]
    /filter Boolean)
    /map square)
    /join "|"))
