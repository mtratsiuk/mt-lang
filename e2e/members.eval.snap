42
4|16|36
